.github/workflows/unsafe-reviewers.yml

```diff
@@ -20,11 +20,24 @@ jobs:
     runs-on: ubuntu-latest
     if: github.event.pull_request.merged != true && github.event.action != 'closed'
     steps:
-      - name: Checkout actions
+      # NOTE: We're checking out both repos to avoid a security vulnerability
+      # Any code that runs in this workflow should be using the checked out base repo to avoid
+      # running code from a potentially malicious PR
+      # https://securitylab.github.com/resources/github-actions-preventing-pwn-requests/
+      # https://nathandavison.com/blog/github-actions-and-the-threat-of-malicious-pull-requests
+      - name: Checkout base repo 
         uses: actions/checkout@v4
         with:
           fetch-depth: 0
+      - name: Checkout head repo
+        uses: actions/checkout@v4
+        with:
+          repository: ${{ github.event.pull_request.head.repo.full_name }}
+          ref: ${{ github.event.pull_request.head.ref }}
+          token: ${{ secrets.GITHUB_TOKEN }}
+          fetch-depth: 0
+          path: head-repo
```

Comment:
Does this put the head-repo in a folder inside the base-repo? Or are their paths separate? Not familiar with github semantics, but we probably don't want to put one repo inside the other.

Reply:
Let me double check this - I think you may be right and if so I'll give this an explicit path and make sure we actually check them out side-by-side.